src/atlantis/manager/rpc/ldap.go

```diff
@@ -264,12 +264,15 @@ func (e *AddTeamAdminExecutor) Authorize() error {
 	if err != nil || !res.IsAdmin {
 		return errors.New("Permission denied")
 	}
-        reql := ManagerListTeamAdminsArg{e.arg.ManagerAuthArg, e.arg.Team}
-        var resl  ManagerListTeamAdminsReply
-        err = NewTask("AddTeamAdmin-ListTeamAdmins", &ListTeamAdminsExecutor{reql, &resl}).Run()
-        if err != nil || len(resl.TeamAdmins) >= maxTeamAdmins {
-                return fmt.Errorf("Maximum of %d admins are allowed per team", maxTeamAdmins)
-        }
+	reql := ManagerListTeamAdminsArg{e.arg.ManagerAuthArg, e.arg.Team}
+	var resl  ManagerListTeamAdminsReply
```

Comment:
The "two spaces" I was referring to were between resl and ManagerListTeamAdminsReply.

Of course, go fmt (or our make fmt shortcut from the root of the repo) will automagically fix that.